app/_locales/en/messages.json

```diff
@@ -5,6 +5,9 @@
   "connectedSitesDescription": {
     "message": "$1 is connected to these sites. They can view your account address."
   },
+  "connectedSitesEmptyDescription": {
+    "message": "$1 is not connected to any sites."
```

Comment:
Ideally any message with a `$1` would have an accompanying description that explains what the `$1` is - to help translators understand the context. It looks like I missed this with the last PR (the message above this one also lacks a description).

Reply:
I'll add a description for this one, yeah

Reply:
Updated and created #8303 as well